docs/tutorials.md

```diff
@@ -8,6 +8,12 @@ nav = [
 ]
 +++
 
+### External Tutorials
```

Comment:
As we discussed IRL, this would be nice as a Note admonition just below the Tutorials heading.